_posts/2022-09-26-changelog-179.markdown

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #179"
+date: 2022-09-26 13:00:00 +0300
+---
+
```

Comment:
I want to [drop](https://github.com/intellij-rust/intellij-rust/pull/9376) support for 2022.1 platform, so let's add a comment that it's the latest plugin version which supports 2022.1 as we usually do